ci/ci.el6/integration_test/multibox/ind-steps/step-box/preconfigure.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+run_cmd "iptables -F"
```

Comment:
What exactly is going on here? If I understand correctly preconfigure is done in chroot before booting the VM so networking shouldn't be up yet and this will not persist after boot.